test/e2e/framework/util.go

```diff
@@ -163,7 +163,6 @@ const (
 	// Minimal number of nodes for the cluster to be considered large.
 	largeClusterThreshold = 100
 
-	// TODO(justinsb): Avoid hardcoding this.
 	awsMasterIP = "172.20.0.9"
```

Comment:
i'd assume that instead of removing the TODO, we might want to remove the `awsMasterIP` constant.
is it still relevant today?
https://github.com/kubernetes/kubernetes/blob/b290e0b7ab292bd5694e2a95e234ac2d43d3a5c0/test/e2e/framework/util.go#L1808

Reply:
This variable seems to be useful for now, as mentioned above, this is for aws. 
But I'm curious why this value is? Let me ask what the main maintainers of test/ think about it?
@sig-testing

Reply:
I don't want to remove this TODO without confirming it by @justinsb.

Reply:
OK, let's wait for his confirmation, whether to delete TODO or complete this TODO。
/cc @justinsb